UPGRADING.rst

```diff
@@ -10,6 +10,17 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register Google service account with SAM during deployment
+================================================================
+
+Google service accounts are now registered with SAM (needed for TDR
+authorization) as a part of the provisioning process during deployment. However,
+previously provisioned credentials are not retroactively registered by default.
+`_select` your personal deployment and run::
+
+     python3 scripts/provision_credentials.py google-key --build azul-ucsc-indexer-"$AZUL_DEPLOYMENT_STAGE"@"$GOOGLE_PROJECT".iam.gserviceaccount.com --register-existing
+
+To register your existing personal deployment's indexer service account.
```

Comment:
We should strive for idempotent, self-healing solutions wherever possible. This current implementation gives us one shot at registering with SAM, at the time the SA is provisioned. If something goes wrong during that, manual intervention is required. It would be more elegant, if running `make deploy` again would fix the issue. The requests to Google and SAM are cheap so I don't mind doing them every time `make deploy` is run.

IIRC, SAM returns a 409 when attempting to register an already registered user so we could make this idempotent and not require this manual intervention.